### DISCUSSION TITLE
Connecting multiple crazyflies with more than one crazyradio

### DISCUSSION THREAD

**Discussion by StewenceNewton** [bitcraze/discussions#1217] [Q&A] [answered]
Good morning,
I want to connect few crazyflies to one crazyradio and remaining for another radio, is it possible to do and if possible please let me know how to do it.

**Comment by knmcguire** [comment, marked as answer]
Yes this is possible! So you have the swarm tutorial with the cflib here: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_swarm_interface/. The only thing that you need to do is change the first number in the URI and put the crazyflies that are on the other radio on a different channel

So from 


```
uris = {
    'radio://0/20/2M/E7E7E7E701',
    'radio://0/20/2M/E7E7E7E702',
    'radio://0/20/2M/E7E7E7E703',
    'radio://0/20/2M/E7E7E7E704',
}
```

to


```
uris = {
    'radio://0/20/2M/E7E7E7E701',
    'radio://0/20/2M/E7E7E7E702',
    'radio://1/40/2M/E7E7E7E703',
    'radio://1/40/2M/E7E7E7E704',
}

```
All on the same channel can probably work out but the performance won't be great I reckon.

**Comment by StewenceNewton** [comment]
Thank you for you response, It is working

**Reply by knmcguire** [reply]
great! I've reopened the discussion and selected my answer as the right answer. This way it will be better visible for other users
